<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>800</width>
      <height>410</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>781</width>
      <height>381</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>隶书</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>商品管理</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>790</width>
        <height>381</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>786</width>
         <height>317</height>
        </rect>
       </property>
       <attribute name="label">
        <string>新品入库</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>2</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>类别：</string>
        </property>
       </widget>
       <widget class="QComboBox" name="newCategoryComboBox">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>2</y>
          <width>96</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>34</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>名称：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="newNameLineEdit">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>34</y>
          <width>142</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>66</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>进价：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="newInputPriceLineEdit">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>66</y>
          <width>159</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="newOutputPriceLineEdit">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>100</y>
          <width>159</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>售价：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>246</x>
          <y>66</y>
          <width>16</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>¥</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>100</y>
          <width>16</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>¥</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>200</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>商品样照</string>
        </property>
       </widget>
       <widget class="QLabel" name="newPictureLabel">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>150</y>
          <width>151</width>
          <height>161</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="newUploadPushButton">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>120</y>
          <width>89</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>上传</string>
        </property>
       </widget>
       <widget class="QTableView" name="commodityTableView">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>0</y>
          <width>381</width>
          <height>321</height>
         </rect>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>25</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>120</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>115</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>数量：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="newCountSpinBox"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>240</y>
          <width>82</width>
          <height>57</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="newPutinStorePushButton">
           <property name="text">
            <string>入库</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="newClearancePushButton">
           <property name="text">
            <string>清仓</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>786</width>
         <height>317</height>
        </rect>
       </property>
       <attribute name="label">
        <string>预售订单</string>
       </attribute>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>类别：</string>
        </property>
       </widget>
       <widget class="QComboBox" name="preCategoryComboBox">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>0</y>
          <width>89</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="newPictureLabel_2">
        <property name="geometry">
         <rect>
          <x>419</x>
          <y>359</y>
          <width>151</width>
          <height>151</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>名称：</string>
        </property>
       </widget>
       <widget class="QComboBox" name="preNameComboBox">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>201</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="prePictureLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>151</width>
          <height>151</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QListWidget" name="sellListWidget">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>-10</y>
          <width>441</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter_3">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>220</y>
          <width>80</width>
          <height>48</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QPushButton" name="preSellPushButton">
         <property name="text">
          <string>出售</string>
         </property>
        </widget>
        <widget class="QPushButton" name="prePlaceOrderPushButton">
         <property name="text">
          <string>下单</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>241</width>
          <height>55</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>单价：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="preOutputPriceLabel">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>¥</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>库存：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="preAmountLabel">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>¥</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>数量：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="preCountSpinBox"/>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>进价：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="preTotalLabel">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>¥</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
